test/_ui-integration/paf/application.spec.js

```diff
@@ -0,0 +1,118 @@
+describe('the journey of a paf application', () => {
+  let testApp;
+  let passStep;
+  let initSession;
+
+  const SUBAPP = 'paf';
+
+  before(() => {
+    testApp = getSupertestApp(SUBAPP);
+    passStep = testApp.passStep;
+    initSession = testApp.initSession;
+  });
+
+  it('goes to the crime-type page', async () => {
+    const URI = '/start';
+    await initSession(URI);
+    const response = await passStep(URI, {});
+    expect(response.text).to.contain('Found. Redirecting to /paf/crime-type');
```

Comment:
Can you explain why it says 'Found. Redirecting...' where is that response from? is it from the response body?

Reply:
Yes this comes from the response.  The expected http status 302 Found happens on a calling
passStep() successfully - it redirects to the next page

Reply:
ah ok, why does it redirect on every page?

Reply:
acceptance tests you usually have to tell it to continue, so I'm wondering how this work

Reply:
the passStep function calls a postUrl function (see the supertest-utilities.js file) which basically acts as the 'continue'/ 'submit' in acceptance tests

Reply:
Ok thanks Rhodine for the explanation, I understand some of it but sounds like I should run through it in debug mode to fully get it.  Nice work